src/libraries/System.DirectoryServices.Protocols/src/System/DirectoryServices/Protocols/Interop/SortKeyInterop.cs

```diff
@@ -0,0 +1,48 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections;
+using System.ComponentModel;
+using System.Diagnostics;
+using System.Text;
+using System.Runtime.InteropServices;
+using System.Security.Principal;
+using System.Runtime.Versioning;
+
+namespace System.DirectoryServices.Protocols
+{
+    internal partial struct SortKeyInterop
+    {
+        private string _name;
```

Comment:
NIT: why do you need the backing fields? Couldn't this all just be:

```c#
        public string AttributeName { get; set; }

        public string MatchingRule  { get; set; }

        public bool ReverseOrder { get; set; }
```

Reply:
Sure, done.